src/Flex.php

```diff
@@ -130,7 +130,12 @@ public function configurePackage(PackageEvent $event): void
 
             $manifest = $recipe->getManifest();
             if (isset($manifest['post-install-output'])) {
-                $this->postInstallOutput = array_merge($this->postInstallOutput, $manifest['post-install-output'], ['']);
+                $postInstallOutput = [];
+                foreach ($manifest['post-install-output'] as $line) {
+                    $postInstallOutput[] = $this->options->expandTargetDir($line);
+                }
+
+                $this->postInstallOutput = array_merge($this->postInstallOutput, $postInstallOutput, ['']);
```

Comment:
instead of building an array `$postInstallOutput` by adding elements in it one by one, and then merging it in `$this->postInstallOutput`, you could add elements in `$this->postInstallOutput` directly in your loop (and then adding the empty string in the array after the loop)